pkg/moq/moq.go

```diff
@@ -13,6 +12,10 @@ import (
 	"os"
 	"strings"
 	"text/template"
+	"golang.org/x/tools/go/loader"
+	"path/filepath"
+	"errors"
+	"path"
```

Comment:
Imports should likely be ordered alphabetically, possibly with the external `golang.org` as its own group below:

```go
import (
    "bytes"
    "errors"
    "fmt"
    "go/ast"
    "go/format"
    "go/parser"
    "go/token"
    "go/types"
    "io"
    "os"
    "path"
    "path/filepath"
    "strings"
    "text/template"

    "golang.org/x/tools/go/loader"
)
```